app/controllers/internal/permissions_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Internal::PermissionsController < Internal::ApplicationController
+  layout "internal"
+
+  def index
+    @users = User.with_role(:admin).union(User.with_role(:super_admin)).union(User.with_role(:single_resource_admin, :any))
```

Comment:
It looks like the [`with_role` scope that rolify gives us](https://github.com/RolifyCommunity/rolify/blob/c05551d8dee3ad85b9c0e450155dbf81887f1a7c/lib/rolify/resource.rb#L12) does not eager load the roles, that might be something we want to do here to avoid N+1 queries in the view when we fetch role information.

Reply:
Given the context here (this is going to return about 5 total users), I think N+1 is the kind of thing we could refactor for later if we touch this code again or it starts to get slow for any reason.

If I'm being overly lazy here let me know. But that's my logic in this kind of situation. 🙃

Reply:
HAHA, I agree with you which is why I approved it, but as the DEV SRE I also feel it's my duty to at least point it out 😊